src/sentry/interfaces/stacktrace.py

```diff
@@ -316,6 +316,8 @@ def to_python(cls, data):
 
         try:
             in_app = validate_bool(data.get('in_app'), False)
+            if package is not None and 'libswiftCore' in package:
```

Comment:
we should instead of doing it here fix this in the native backend. It's already fixed on a different branch so I would remove the check here entirely.